quit/git.py

```diff
@@ -22,15 +22,30 @@
 
 
 class Repository(object):
-    def __init__(self, path, **params):
-        origin = params.get('origin', None)
-
+    """The Quit class for wrapping a git repository.
+
+    TODO:
+    - There is no possibility to set remotes on a Quit Repository object.
+    """
+
+    def __init__(self, path, origin=None, create=False, garbageCollection=False):
+        """Initialize a quit repo at a given location of the filesystem.
+
+        Keyword arguments:
+        path -- the location of an existing git repository or where the repository should be created
+        origin -- if the repository does not exist and origin is given the origin repository is
+                  cloned to the given path (default: None)
+        create -- boolean whether to create a new repository if the cpecified path is empty
+                  (default: False)
+        garbageCollection -- boolean whether to activate the garbage collection on the git
+                  repository (default: False)
+        """
         self.callback = QuitRemoteCallbacks()
 
         try:
             self._repository = pygit2.Repository(path)
         except KeyError:
```

Comment:
Do you know if KeyError is raised in your case?

Reply:
This is not subject of this PR